discover/faq.md

```diff
@@ -12,978 +12,157 @@ Hopefully, you'll find the answers here! If not then please don't hesitate to re
-1. Ocean [Templates](https://oceanprotocol.com/templates). 
-2. Introducing Ocean Templates [blog post](https://blog.oceanprotocol.com/introducing-ocean-templates-tools-to-build-next-generation-web3-dapps-67b66e519eb1).
-3. Ocean Token Model 2023 [blog post](https://blog.oceanprotocol.com/ocean-token-model-2023-2f306932f34a).
-  
+The Ethereum network has the highest liquidity for $OCEAN.  
 </details>
 
 <details>
+<summary>Can the Ocean tech stack be used without utilizing the OCEAN token?</summary>
 
-<summary>Is it possible to sell models on Ocean Protocol?</summary>
+All Ocean modules and components are open-source and freely available to the community. Developers can change the default currency from OCEAN to a different one for their dApp.
 
-Yes, you can sell any type of data. Start learning about us through the [documentation](https://docs.oceanprotocol.com/).
-  
 </details>
 
-
 <details>
+<summary>How does the ecosystem and the Ocean token benefit from the usage of the open-source tech stack when transactions can be paid in any currency?</summary>
 
-<summary>Is it true that Mercedes Benz has partnered with Ocean?</summary>
+You are right, all Ocean modules and components are open-source and freely available to the community. Developers can change the default currency from OCEAN to a different one for their dApp. But, there is a community fee involved in all transactions hapening in the ecosystem.
 
-Yes 🫢, [here](https://blog.oceanprotocol.com/acentrik-a-decentralized-data-marketplace-for-enterprises-built-on-ocean-protocol-is-now-in-7fb7371e57d4) are a few details.
-  
+veOCEAN on the other hand, follows the model of other ve tokens, where it can be used for passive and active staking. By locking up your $OCEAN to veOCEAN, you automatically receive passive staking rewards. You can also earn active staking rewards by assigning your veOCEAN directly on datasets or through a proxy (psdnOCEAN), who deploy your veOCEAN at no risk, in order to gain a share of active rewards. Active rewards are based on transaction volume on specific data assets.
```

Comment:
Delete this paragraph "veOCEAN on the other hand...".

Why: out of place wrt the Q.